components/organisms/animations/AnimFromCliCodeblock.vue

```diff
@@ -114,23 +114,25 @@
     </div>
     <div class="fromcli__display">
       <div class="fromcli__display_loading">
-        <img loading="lazy" class="fromcli__display__logo" src="~/static/img/home/learn/master/logo.svg" />
+        <img loading="lazy" class="fromcli__display__logo" src="~/static/img/home/learn/master/logo.svg" alt="" />
```

Comment:
We possibly should set `role="presentation"` as well here ☺️

Reply:
I think we can place it on `div.fromcli__display` so it applies to all of the child images as well.

Reply:
I applied `role="img"` to both of the animation components one level up. I'm not sure if we need to set the role on this specifically. Or if we should change it to `presentation`.